WD-Firmware/demo/demo_platform_al.c

```diff
@@ -30,6 +30,10 @@
 
 #ifdef D_HI_FIVE1
    #include <unistd.h>
+#endif
+
+#ifdef D_NEXYS_A7
+   #include "printf.h"
 #else
    PRE_COMPILED_MSG("no platform was defined")

```

Comment:
I think you'll faill here when building hifive1
should be 
#ifdef D_HI_FIVE1
   #include <unistd.h>
#elif defined(D_NEXYS_A7)
   #include "printf.h"
#else
   PRE_COMPILED_MSG("no platform was defined")
#endif

Reply:
good catch, thanks.
Fiexed